shiny/examples/nav_show/app.py

```diff
@@ -0,0 +1,48 @@
+from shiny import *
```

Comment:
```suggestion
```suggestion
from shiny import App, reactive, ui
```
```